examples/metric-receiver/main.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"storj.io/storj/pkg/process"
+	"storj.io/storj/pkg/telemetry"
+)
+
+var (
+	addr = flag.String("addr", ":9000", "address to listen for metrics on")
+)
+
+func main() {
+	process.Must(process.Main(process.ServiceFunc(run)))
```

Comment:
should we just handle the error in Process ?

Reply:
i'm okay with that